docker-images/artifacts/kafka-thirdparty-libs/cc/pom.xml

```diff
@@ -17,8 +17,21 @@
 
     <properties>
         <cruise-control.version>2.5.142</cruise-control.version>
+        <log4j.version>2.17.2</log4j.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-bom</artifactId>
+                <version>${log4j.version}</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

Comment:
BOMs are known to cause problems in creating dependency conflicts. I do not think this should be here. We should stick with the exact version of the dependency below.

Reply:
+1